packages/sdk-client-v3/src/utils/validate.ts

```diff
@@ -19,11 +19,6 @@ export function validateHttpOptions(options: HttpMiddlewareOptions) {
     throw new Error(
       'An `httpClient` is not available, please pass in a `fetch` or `axios` instance as an option or have them globally available.'
     )
-
-  if (options.timeout && !options.getAbortController)
```

Comment:
See: https://github.com/commercetools/commercetools-sdk-typescript/blob/68ad0e322b355cdee28e3114c95e60d0b9191df0/packages/sdk-client-v3/src/middleware/create-http-middleware.ts#L167

Reply:
This was added in nodejs v15 [see docs](https://nodejs.org/api/globals.html#class-abortcontroller) nodejs version prior doesn't have this global object, hence the check we performed on line 24.